install/kubernetes/cilium/files/hubble/dashboards/hubble-dashboard.json

```diff
@@ -3303,7 +3303,22 @@
   "style": "dark",
   "tags": [],
   "templating": {
-    "list": []
+    "list":
+      {
+        "current": {},
+        "hide": 0,
+        "includeAll": false,
+        "label": "Prometheus",
+        "multi": false,
+        "name": "DS_PROMETHEUS",
+        "options": [],
+        "query": "prometheus",
+        "queryValue": "",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
+      }
```

Comment:
there is a little bug here. this should be wrapped in square brackets since it's supposed to be a list of objects :)

Reply:
yep, currently this dashboard shows "Dashboard init failed" on load

Reply:
thanks